drivers/sensors/cirque_pinnacle.c

```diff
@@ -241,6 +293,8 @@ void cirque_pinnacle_init(void) {
     RAP_Write(Z_IDLE_COUNT, 5);
 
     cirque_pinnacle_set_adc_attenuation(CIRQUE_PINNACLE_ATTENUATION);
+    // Perform manual calibration after setting ADC attenuation
```

Comment:
"manual calibration" sounds like the user needs to tweak a potentiometer or something. This is in fact more an "auto-calibration" or "self-calibration" routine, right?

Reply:
Fair enough, how about rewording it as "Force a calibration after setting ADC attenuation"? Borrows terminology from GT-AN-090624 section 1.2. I would call it "force compensation" but the bit itself is called "calibrate".

Reply:
_perform/trigger/force_ are all 👍